.env.example

```diff
@@ -46,6 +46,7 @@ WEB_AGENT_LLM="openai"
 CHART_GENERATOR_LLM="openai"
 ROUTER_LLM="openai"
 FILE_AGENT_LLM="openai"
+SUGGESTIONS_LLM="openai"
```

Comment:
Could you add a note about putting these variables into your .env file in the PR description